data_quality/tasks/assess_performance.py

```diff
@@ -17,10 +17,18 @@ class PerformanceAssessor(Task):
 
     """A Task runner to assess and write the performance of publishers for each
        period.
+
+    Note the distiction between a source's creation period
+     (referenced here as "period") and the period the content of a source
+     reffers to (referenced here as "relevance_period").
     """
 
     def __init__(self, *args, **kwargs):
         super(PerformanceAssessor, self).__init__(*args, **kwargs)
+        self.assess_timeliness = self.config['assess_timeliness']
+        timeliness_params = self.config['timeliness']
+        self.timeliness_period = timeliness_params.get('timeliness_period', '1')
+        self.max_empty_period = timeliness_params.get('max_empty_period', '10')
```

Comment:
It is the maximum percent of lines that can have empty `relevance_period` but it wasn't clear from the name. I changed it to `max_empty_relevance_period`.